plugins/destination/snowflake/guide.md

```diff
@@ -0,0 +1,22 @@
+Create Stage:
+
+JSON
+
+```
+create or replace file format myjsonformat
+  type = 'JSON'
+  strip_outer_array = true;
```

Comment:
This should be `false` if we're writing LDJSON as we are in the file plugin.

Reply:
Good catch. I think it doesn't have any effect but removed.